x-pack/plugin/eql/qa/common/src/main/java/org/elasticsearch/test/eql/DataLoader.java

```diff
@@ -130,6 +134,10 @@ private static String readMapping(URL resource) throws IOException {
         }
     }
 
+    private static CharSequence randomOf(String...values) {
+        return main ? values[0] : ESRestTestCase.randomFrom(values);
+    }
+
```

Comment:
I've sneaked this in since it affects running DataLoader#main instead of from a test.